app/src/main/res/layout/fragment_timeline.xml

```diff
@@ -29,8 +29,9 @@
 
     <com.keylesspalace.tusky.view.BackgroundMessageView
         android:id="@+id/statusView"
-        android:layout_width="wrap_content"
```

Comment:
With wrap_content (esp. the height) this didn't really work out for me. It centers things somehow...

However I want something on the top of the screen.